packages/admin/src/App/Home/Products.js

```diff
@@ -2,15 +2,18 @@ import React from "react";
 import { getProducts, deleteProduct } from "./service.js";
 import { Redirect } from "react-router-dom";
 
-import { Table, Avatar, Button, Divider } from "antd";
+import { Table, Avatar, Button, Divider, Modal } from "antd";
 import { Link } from "react-router-dom";
 import "./Products.scss";
 
 class Products extends React.Component {
   state = {
     products: { data: [] },
     loading: false,
-    redirect: ""
+    redirect: "",
+    isModalOpen: false,
+    isDeletingProd: false,
+    selectedProd: ""
```

Comment:
Got it, just that I think `isDeletingProd` is still needed in order to display the loading during the delete request.
I will also change #3 accordingly because they are implemented in a similar way.